elastic/kibana
# [Cloud Security] Converts graph visualization from experimental feature to the advanced settings

## Summary

During 8.17 development cycle, we introduced a new feature behind the experimental flag. 
A user could not opt-in and use it on serverless either. Therefore this change does not introduce a breaking change.

![Screenshot 2024-11-11 at 20 39 39](https://github.com/user-attachments/assets/cca18f4e-3c7a-4446-89f3-319b0c863af1)

### How to test 

1. Toggle feature flag
2. Load mock data

```bash
node scripts/es_archiver load x-pack/test/cloud_security_posture_functional/es_archives/logs_gcp_audit \ 
  --es-url http://elastic:changeme@localhost:9200 \
  --kibana-url http://elastic:changeme@localhost:5601

node scripts/es_archiver load x-pack/test/cloud_security_posture_functional/es_archives/security_alerts \
  --es-url http://elastic:changeme@localhost:9200 \
  --kibana-url http://elastic:changeme@localhost:5601
```

3. Go to the alerts page
4. Change the query time range to show alerts from the 13th of October 2024 (**IMPORTANT**)
5. Open the alerts flyout
6. Scroll to see the graph visualization : D


**Related PRs**
- https://github.com/elastic/kibana/pull/196034
- https://github.com/elastic/kibana/pull/195307

### Checklist

Delete any items that are not applicable to this PR.
